llvm/lib/Target/AArch64/GISel/AArch64CallLowering.cpp

```diff
@@ -525,10 +527,10 @@ static void handleMustTailForwardedRegisters(MachineIRBuilder &MIRBuilder,
 
 bool AArch64CallLowering::fallBackToDAGISel(const MachineFunction &MF) const {
   auto &F = MF.getFunction();
-  if (F.getReturnType()->isScalableTy() ||
+  if (!EnableSVEGISel && (F.getReturnType()->isScalableTy() ||
```

Comment:
Are scalable return types enabled now?

Reply:
No. I'm following the same idea from https://github.com/llvm/llvm-project/pull/92130#discussion_r1600378977 to enable SVE only with a debug option